lib/ClientSai.cpp

```diff
@@ -1060,6 +1060,27 @@ sai_status_t ClientSai::queryStatsCapability(
     return waitForQueryStatsCapabilityResponse(stats_capability);
 }
 
+sai_status_t ClientSai::queryStatsStCapability(
```

Comment:
did we add the notification dummy entry?

Reply:
Not yet, maybe next commit.